workspaces/jupyterlab-r/install.sh

```diff
@@ -3,9 +3,9 @@
 set -ex
 
 # Install all R dependencies.
-xargs mamba install --yes r-base==4.4.3 r-irkernel==1.3.2
+xargs mamba install --yes r-base==4.5.1 r-irkernel==1.3.2
```

Comment:
Can we not do a `mamba install` from the `requirements.txt`? That seems more conventional for listing packages, and dependabot may understand that format better.

Reply:
Ah I see, you can ignore this.